config/locales/client.en.yml

```diff
@@ -56,6 +56,8 @@ en:
       assign_post_modal:
         title: "Assign Post"
         description: "Enter the name of the user you'd like to assign this post"
+      assign_bulk_modal:
+        description: "Enter the name of the user you'd like to assign these topics"
```

Comment:
Looks good but just want to confirm that it should be "topics" not "posts"?

Reply:
Yes the "topics" is correct here since the bulk assign action can only be performed on topics via bulk topic action modal.

Thanks for the review! :)

Reply:
Yup okay just saw the topic.